sqlite_utils/db.py

```diff
@@ -64,13 +64,46 @@
         "least_common",
     ),
 )
-ForeignKey = namedtuple(
-    "ForeignKey", ("table", "column", "other_table", "other_column")
-)
 Index = namedtuple("Index", ("seq", "name", "unique", "origin", "partial", "columns"))
 Trigger = namedtuple("Trigger", ("name", "table", "sql"))
 
 
+class ForeignKey(ForeignKeyBase):
```

Comment:
It looks like `ForeignKeyBase` is missing - I added this and the tests started passing again:
```python
ForeignKeyBase = namedtuple(
    "ForeignKey", ("table", "column", "other_table", "other_column")
)
```